frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte

```diff
@@ -84,7 +84,7 @@
 			</h1>
 		</div>
 		<h5 class="mt-4 ml-1 text-lg">Author: {data.report.ownerName}</h5>
-		<hr class="mt-4 bg-grey-dark" />
+		<hr class="mt-4 text-grey-light" />
```

Comment:
Should we make this a border on the <p> element below? Like this we have these two lines even if there is no associated project.

Reply:
The other one is in the component so will only render one.